components/automate-ui/src/app/pages/project/list/project-list.component.html

```diff
@@ -14,37 +14,49 @@
+      <ng-container *ngIf="(sortedProjects$ | async)?.length > 0">
+        <chef-toolbar>
+          <app-authorized [allOf]="['/iam/v2beta/projects', 'post']">
+            <chef-button id="create-button" primary (click)="openCreateModal()">Create Project</chef-button>
+          </app-authorized>
+        </chef-toolbar>
+        <app-authorized [allOf]="['/iam/v2beta/projects', 'get']">
+          <chef-table>
+            <chef-thead>
+              <chef-tr>
+                <chef-th>Name</chef-th>
+                <chef-th>Type</chef-th>
+                <chef-th class="controls"></chef-th>
+              </chef-tr>
+            </chef-thead>
+            <chef-tbody>
+              <chef-tr *ngFor="let project of sortedProjects$ | async">
+                <chef-td>
+                  <a [routerLink]="['/settings/projects', project.id]">{{ project.name }}</a>
+                </chef-td>
+                <chef-td>{{ project.type | iamType }}</chef-td>
+                <chef-td class="controls">
+                  <app-authorized [allOf]="['/iam/v2beta/projects/{id}', 'delete', project.id]">
+                    <chef-control-menu>
+                      <chef-option (click)="startProjectDelete(project)">Delete Project</chef-option>
+                    </chef-control-menu>
+                  </app-authorized>
+                </chef-td>
+              </chef-tr>
+            </chef-tbody>
+          </chef-table>
         </app-authorized>
-      </chef-toolbar>
-      <app-authorized [allOf]="['/iam/v2beta/projects', 'get']">
-        <chef-table>
-          <chef-thead>
-            <chef-tr>
-              <chef-th>Name</chef-th>
-              <chef-th>Type</chef-th>
-              <chef-th class="controls"></chef-th>
-            </chef-tr>
-          </chef-thead>
-          <chef-tbody>
-            <chef-tr *ngFor="let project of sortedProjects$ | async">
-              <chef-td>
-                <a [routerLink]="['/settings/projects', project.id]">{{ project.name }}</a>
-              </chef-td>
-              <chef-td>{{ project.type | iamType }}</chef-td>
-              <chef-td class="controls">
-                <app-authorized [allOf]="['/iam/v2beta/projects/{id}', 'delete', project.id]">
-                  <chef-control-menu>
-                    <chef-option (click)="startProjectDelete(project)">Delete Project</chef-option>
-                  </chef-control-menu>
-                </app-authorized>
-              </chef-td>
-            </chef-tr>
-          </chef-tbody>
-        </chef-table>
-      </app-authorized>
+      </ng-container>
+      <ng-container *ngIf="(sortedProjects$ | async)?.length === 0">
+        <div class="empty-case-container">
+          <h1>Create the first project to get started!</h1>
+        </div>
+        <div class="empty-case-container">
+          <app-authorized [allOf]="['/iam/v2beta/projects', 'post']">
```

Comment:
This will look rather odd if the user is not authorized, having the friendly greeting (L52) but no button! Seems like the `app-authorized` for this case needs to move out to the `ng-container`... but then the screen will be blank.
Suggest a UX intervention -- @susanev ?

Reply:
good point! hmm, i think we need a totally separate message in that case. i need some time to think about it. think we could come back to that later? or should we solve that now?

Reply:
@tylercloke if its possible to do in this pr, here is a design for the case @msorens describes.
https://zpl.io/VkpeWKK

Reply:
@msorens Do we have a way to say `not authorized` in the HTML? Not that I'm aware of 🤔. 

```
        <app-authorized [allOf]="['/iam/v2beta/projects', 'post']">
          <div class="empty-case-container">
            <h1>Create the first project to get started!</h1>
          </div>
          <div class="empty-case-container">
              <chef-button primary (click)="openCreateModal()">Create Project</chef-button>
          </div>
        </app-authorized>
        <ELSE?>
```

Reply:
Copying my reply here as well, just for completeness:
Build an `authorizedChecker`, call its `setPermissions`, subscribe to its `isAuthorized$` observable, and plug that into an `ngIf`. :boom: See working example in landing.component.ts.